source/client/cl_game.c

```diff
@@ -22,7 +22,7 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 #include "cin.h"
 #include "../qcommon/asyncstream.h"
 
-static cgame_export_t *cge;
+cgame_export_t *cge;
```

Comment:
This is probably not a good idea.

Reply:
I'm not sure what other move to take? I need to give ui access to the blocklist in cge

Reply:
Something like `size_t ( *ML_GetMapByNum )( int num, char *out, size_t size );` could be used for enumerator,
or more like `int ( *FS_GetFileList )( const char *dir, const char *extension, char *buf, size_t bufsize, int start, int end );`